include/zephyr/canbus/isotp.h

```diff
@@ -234,7 +234,7 @@ struct isotp_recv_ctx;
  * When calling this routine, a filter is applied in the CAN device, and the
  * context is initialized. The context must be valid until calling unbind.
  *
- * @param ctx     Context to store the internal states.
+ * @param rctx    Context to store the internal states.
```

Comment:
I must admit I don't think using `rctx` makes the context more clear. How about using `rx_ctx`?

Reply:
I think `rctx` is a good compromise between variable length and clarity. `rx_ctx` is not suitable IMO, as the `rx` prefix is used for the reception on single CAN frame level. We have an `rx_addr` and a `tx_addr` in both `send_ctx` and `recv_ctx`, where one of the addresses is only used for flow control, depending on the direction of communication.

I'd like to keep the terms `recv` and `send` for the higher-level ISO-TP data transfer directions. The only alternative would be `send_ctx` and `recv_ctx`, which makes the variables much longer. I prefer keeping the suggested short `rctx` and `sctx`.